### PR TITLE
Allow Info.plist override by environment variable.

### PR DESCRIPTION
**Pull request by shepting**
## Goal
This is the only configuration option that doesn't yet have an environment variable that can be used to set the value. If you need to call `upload_symbols_to_bugsnag` in many places it can dirty up your `Fastfile`.

Screen Shot 2020-06-25 at 4 26 04 PM

This change would allow one `ENV` setting to apply to all of those callsites.

## Design
This is consistent with the rest of the existing options.

## Changeset
Added a new environment variable.

## Tests
Not tested at all.

### Alternative Approaches
Not making this change would leave `config_file:` argument as the only way to override the `Info.plist` file to be used.

## Review

For the submitter, initial self-review:

- [ ] Commented on code changes inline explain the reasoning behind the approach
- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] A changelog entry was added for the goal of this pull request
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?
- This pull request is ready for:
  - [x] Initial review of the intended approach, not yet feature complete
  - [x] Structural review of the classes, functions, and properties modified
  - [x] Final review

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency across platforms for structures or concepts added or modified
- [ ] Consistency between the changeset and the goal stated above
- [ ] Internal consistency with the rest of the library - is there any overlap between existing interfaces and any which have been added?
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Performance and complexity - are there any cases of unexpected O(n^3) when iterating, recursing, flat mapping, etc?
- [ ] Concurrency concerns - if components are accessed asynchronously, what issues will arise
- [ ] Thoroughness of added tests and any missing edge cases
- [ ] Idiomatic use of the language

### CODE REVIEW

File: tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/upload_symbols_to_bugsnag.rb [hunk 1]
```diff
@@ -112,6 +112,7 @@ def self.available_options
                                        default_value: Dir::pwd,
                                        optional: true),
           FastlaneCore::ConfigItem.new(key: :config_file,
+                                       env_name: "BUGSNAG_INFO_PLIST",
```
Comment (shepting) [inline tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/upload_symbols_to_bugsnag.rb:115]: This was the only `ConfigItem` without and `env_name` option.

**Comment by phillipsam** [comment]
Hi @shepting thanks for the PR! We will look into this.

**Comment by xander-jones** [comment]
Hey @shepting, this has been addressed in [v2.0.0](https://github.com/bugsnag/bugsnag-dsym-upload/releases/tag/v2.0.0). You can now set a `config_file` via environment variable `BUGSNAG_CONFIG_FILE`. Note you can also override the API key/release stage that's extracted from the `config_file` by setting the input in `Fastfile`, or the respective environment variable.

**Comment by shepting** [comment]
Thank you!
